### ISSUE TITLE
integration with dns66?

### ISSUE THREAD

**Issue by 0-kaladin** [AdAway/AdAway#814]
What do you guys think?  
https://github.com/julian-klode/dns66/issues/39

@PerfectSlayer @sanjay900

**Comment by sanjay900** [comment]
That would be pretty cool, since root access is getting pretty hard to patch around.

**Comment by sanjay900** [comment]
Although it currently does have bugs with nougat, which could be a problem

**Comment by PerfectSlayer** [comment]
Hi @0-kaladin and thanks for including me to the debate.

I think it's a good idea. 
AdAway has a strong and good image for the users but root becomes more and more complex and annoying (OTA and Android Pay blocked meanwhile Android getting more and more polish and complete).
So it could be great to offer basic ad block feature without root access so people could keep using their beloved adblocker and progressively stop rooting their devices if it's the last feature they use that requires it.

In short, I'm in favor of merging with dns66 and I'll proud to help you to deal with the code bases 😉

**Comment by sanjay900** [comment]
This does seem like a good idea, I'm happy to help.

**Comment by sanjay900** [comment]
Is this going ahead?

**Comment by PerfectSlayer** [comment]
I didn't dare to ask ! It looks like too much to an ETA request 😅
